targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F407xE/device/system_stm32f4xx.h

```diff
@@ -0,0 +1,125 @@
+/**
+  ******************************************************************************
+  * @file    system_stm32f4xx.h
+  * @author  MCD Application Team
+  * @brief   CMSIS Cortex-M4 Device System Source File for STM32F4xx devices.       
+  ******************************************************************************  
+  * @attention
+  *
+  * <h2><center>&copy; COPYRIGHT(c) 2017-2019 STMicroelectronics</center></h2>
+  *
+  * SPDX-License-Identifier: Apache-2.0
```

Comment:
this is BSD not Apache

Reply:
Some other files as well

Reply:
@0xc0170 Thanks for point this out.  Fixed.